### PR TITLE
Make plugin compatible for 2021.2

### LINKED ISSUE #21: Gradle?
i think the recommended approach for intellij plugin dev is by using gradle

### PR DESCRIPTION
**Pull request by MaXal**
This closes: #21  and #23 besides it includes CI helpers that help resolve and provide completion for CI 3 magic.

### CODE REVIEW

**Comment by MaXal** [comment]
@martynassateika Please let me know if you're not interested in supporting the plugin anymore. In this case, we (PhpStorm team) will publish and will maintain the fork (which of course will reference you as an author of the original plugin).

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
 
   <description><![CDATA[
       A PhpStorm plugin for CodeIgniter development.
     ]]></description>
 
   <change-notes><![CDATA[
+      0.6 (2021-08-02):
+      <ul>
+          <li>Built-in CodeIgniter helpers</li>
+          <li>Fixed: Compatibility issues with PhpStorm 2021.2 (thanks @ivothgle!)</li>
```
Comment (martynassateika) [inline src/main/resources/META-INF/plugin.xml]:
```suggestion
          <li>Fixed: Compatibility issues with PhpStorm 2021.2 (thanks @MaXal!)</li>
```

**Review by martynassateika — APPROVED** [review]
Thank you for the PR, Maxim!

Apologies for the late response, and my lack of support for the plugin lately. Due to a recent move and no access to my main machine I don't have an easy way to build the plugin. I am more than happy for JetBrains to take over, as I am well aware of people using this plugin and frustrated with its incompatibility with recent versions of PhpStorm.

Please let me know if you need anything else from me. Happy to merge this PR so you can fork it after, just one tiny suggestion (wrong person credited).

EDIT: Actually, let me just sort this through the GitHub UI 🙂

**Comment by MaXal** [comment]
@martynassateika Thank you a lot! We will fork it to JetBrains account and republish it with all references pointing to your work!

**Pull request merged**